app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] One of the requirements for this project milestone is to implement the confirmation of email for a user. Kindly implement this feature. 👍 😎 

![image](https://user-images.githubusercontent.com/77106962/226987995-090d066a-c447-4050-a7e4-482c06339756.png)

#### Hint: Add the `:confirmable` attribute to the devise modules in your `app/models/user.rb` file